sycl/doc/extensions/SubGroupMask/SubGroupMask.asciidoc

```diff
@@ -74,25 +74,25 @@ the specification.
 
 === Ballot
 
-The `group_ballot` algorithm converts a Boolean condition from each work-item
-in the group into a group mask.  Like other group algorithms, `group_ballot`
+The `sub_group_ballot` algorithm converts a Boolean condition from each work-item
```

Comment:
When we discussed renaming yesterday I hadn't actually thought about what to call the free-function.

@rolandschulz, @gmlueck: Do you think we should call this `sub_group_ballot`, or is it okay for `group_ballot` to return a `sub_group_mask` when handed a `sub_group`, leaving the possibility open for `group_ballot` to return something else when handed a `group`?

Reply:
If we keep the `Group` type as a template parameter, then I think the function should be named `group_ballot()`.  Alternatively, if we decide to name the function `sub_group_ballot()`, then we should hard-code the parameter type to `sub_group`, rather than using a template parameter.  Either seem OK to me.